source/unified-test-format/unified-test-format.rst

```diff
@@ -1377,6 +1379,11 @@ The structure of this object is as follows:
     <../server-discovery-and-monitoring/server-discovery-and-monitoring.rst#servertype>`__
     for a list of valid values.
 
+.. _expectedEvent_topologyDescriptionChangedEvent:
+
+- ``topologyDescriptionChangedEvent``: Optional object. If present, this object MUST be an
+  empty document as all fields in this event are non-deterministic.
```

Comment:
```suggestion
- ``topologyDescriptionChangedEvent``: Optional object. If present, this object
  MUST be an empty document as no assertions are currently supported for
  `TopologyDescriptionChangedEvent <../server-discovery-and-monitoring/server-discovery-and-monitoring-monitoring.rst#events>`__ fields.
```

"all fields in this event are non-deterministic" is specific to some of the CMAP events but it's not applicable to TopologyDescriptionChangedEvent. This text is more accurate.